files/.condarc

```diff
@@ -1,8 +1,12 @@
 channels:
   - conda-forge
   - r
-  - bioconda
+  - moj-analytical-services
```

Comment:
Do they use anything in the `bioconda` channel at the moment? (in theory the can always update their own `.condarc` and add channels anyway right?)

Reply:
i don't think so, I've inspected the existing environment.ymls and nothing is coming from bioconda